presto-main/src/main/java/com/facebook/presto/metadata/ViewInfo.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.metadata;
+
+import java.util.Optional;
+
+import static java.util.Objects.requireNonNull;
+
+public final class ViewInfo
+{
+    private final ViewDefinition definition;
+    private final Optional<String> comment;
```

Comment:
Sorry I might be missing some context, but why not just add comment to ViewDefinition?

Reply:
ViewDefinition is an opaque serialized object given to the connector. We want the comment to be a connector visible field.